nixos/modules/image/repart.nix

```diff
@@ -90,8 +90,8 @@ in
     };
 
     package = lib.mkPackageOption pkgs "systemd-repart" {
-      default = "systemd";
-      example = "pkgs.systemdMinimal.override { withCryptsetup = true; }";
+      default = [ "buildPackages" "systemd" ];
```

Comment:
Please add a comment why this use buildPackages, i.e. because its needed for cross compiling.

Reply:
added